src/histogram-date-range.ts

```diff
@@ -38,10 +39,10 @@ const tooltipFontSize = css`var(--histogramDateRangeTooltipFontSize, 1.1rem)`;
 
 type SliderId = 'slider-min' | 'slider-max';
 
-interface HistogramInputData {
-  minDate: string;
-  maxDate: string;
-  bins: number[];
+class HistogramInputData {
+  minDate = '';
+  maxDate = '';
+  bins: number[] = [];
```

Comment:
It looks like you're checking for the existence of these properties down below, does that mean they can be optional? If so, I would mark them optional so typescript can enforce them.

Reply:
they would only be missing if someone passed in an invalid data object. (see the test file for examples). but if we don't have valid data, then we don't try to access `minDate` etc so I didn't want to make them optional. Does this make sense?